core/src/main/java/com/sequenceiq/distrox/v1/distrox/StackOperations.java

```diff
@@ -258,6 +258,14 @@ public FlowIdentifier putStart(NameOrCrn nameOrCrn, Long workspaceId) {
         return stackCommonService.putStartInWorkspace(nameOrCrn, workspaceId);
     }
 
+    public void rotateSaltPassword(NameOrCrn nameOrCrn, Long workspaceId) {
+        Stack stack = stackService.getByNameOrCrnAndWorkspaceIdWithLists(nameOrCrn, workspaceId);
```

Comment:
is the mdccontext already filled here? if not you might have to initialize with stack

Reply:
Nice catch, fixed it.